docs/welcome.md

```diff
@@ -0,0 +1,30 @@
+---
+id: welcome
+title: Welcome to Hubs
+sidebar_label: Welcome
+---
+
+## What is Hubs?
+Hubs is a social VR platform that runs right in your browser. With Hubs you can create your own virtual space with a single click and invite others to join using a URL. No installation process. No app store. No gatekeepers.
+
+Watch videos with friends, give presentations, or meet with colleagues remotely. Hubs makes it easy to connect with others and share images, videos, 3D models, and more. 
+
+Hubs works across platforms. Got a VR headset? Awesome! If not, you can use your computer or mobile device.
+
+Hubs is open source and customizable. You can choose how you represent yourself by [uploading your own avatar](insert-avatar-url), or build your own world using Hubs' environment editor [Spoke](https://hubs.mozilla.com/spoke).
+
+![Hubs Image](../img/hubs.png)
+
+
+### Who is Hubs for?
+
+Hubs is for everyone! It's a great way to bring communities together in a virtual space. Host a conference, teach a class, create art, or just hang out with friends. Because of Hubs' spatialized audio you can have conversations with everyone together or break out into smaller groups, just like you can in person. 
+
+
+### Key Features
+
+* Works accross platforms
```

Comment:
nitpick - typo extra c

I would say something like "works across desktop, mobile, and VR platforms" to be explicit